Source/WebCore/dom/InlineStyleSheetOwner.cpp

```diff
@@ -156,36 +153,37 @@ inline bool isValidCSSContentType(const AtomString& type)
 void InlineStyleSheetOwner::createSheet(Element& element, const String& text)
 {
     ASSERT(element.isConnected());
-    Document& document = element.document();
-    if (m_sheet) {
-        if (m_sheet->isLoading() && m_styleScope)
-            m_styleScope->removePendingSheet(element);
+    Ref document = element.document();
+    if (RefPtr sheet = m_sheet) {
+        if (sheet->isLoading() && m_styleScope)
+            CheckedRef { *m_styleScope }->removePendingSheet(element);
         clearSheet();
     }
 
     if (!isValidCSSContentType(m_contentType))
         return;
 
-    ASSERT(document.contentSecurityPolicy());
-    const ContentSecurityPolicy& contentSecurityPolicy = *document.contentSecurityPolicy();
-    if (!contentSecurityPolicy.allowInlineStyle(document.url().string(), m_startTextPosition.m_line, text, CheckUnsafeHashes::No, element, element.nonce(), element.isInUserAgentShadowTree())) {
+    ASSERT(document->contentSecurityPolicy());
```

Comment:
Maybe this should be a RELEASE_ASSERT, since we go on and dereference the potentially nullptr value later on?

Reply:
A null dereference is a safe / very identifiable crash already though.